app/src/main/res/layout/activity_forum_categories.xml

```diff
@@ -0,0 +1,288 @@
+                android:layout_height="150dp"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="20dp"
+                android:layout_marginEnd="20dp"
+                android:layout_marginBottom="20dp"
+                android:background="@drawable/red_background">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-condensed-medium"
+                    android:padding="25dp"
+                    android:text="@string/general_medicine"
+                    android:textSize="25sp" />
+
+                <Button
+                    android:id="@+id/generalist_expand_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentBottom="true"
+                    android:background="#00000000"
+                    android:onClick="goToForum"
+                    android:padding="20dp"
+                    android:text="@string/expand_forum" />
+
+
+            </RelativeLayout>
+
+            <ImageView
+                android:layout_width="100dp"
+                android:layout_height="160dp"
+                android:layout_alignParentEnd="true"
+                android:layout_marginTop="-30dp"
+                android:src="@drawable/ic_generalist" />
+        </RelativeLayout>
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="190dp">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="150dp"
+                android:background="@drawable/red_background"
+                android:layout_margin="20dp">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-condensed-medium"
+                    android:padding="25dp"
+                    android:text="@string/cardiology"
+                    android:textSize="25sp" />
+
+                <Button
+                    android:id="@+id/cardio_expand_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentBottom="true"
+                    android:layout_marginBottom="0dp"
+                    android:background="#00000000"
+                    android:padding="20dp"
+                    android:text="@string/expand_forum"
+                    android:onClick="goToForum"/>
+
+
+            </RelativeLayout>
+
+            <ImageView
+                android:layout_width="100dp"
+                android:layout_height="150dp"
+                android:layout_alignParentEnd="true"
+                android:layout_marginTop="-26dp"
```

Comment:
Is this margin meant to be negative?

Reply:
Yes ! Weird but I think the origin is not the corner of the device but rather the red card's corner